cli/running/process.go

```diff
@@ -420,3 +421,63 @@ func (process *Process) Log(follow bool, n int) error {
 
 	return nil
 }
+
+func (process *Process) Clean() ProcessRes {
+	res := ProcessRes{
+		ProcessID: process.ID,
+	}
+
+	pathsToDelete := []string{
+		process.logFile,
+		process.workDir,
+		process.consoleSock,
+		process.notifySockPath,
+		// PID file isn't deleted
+		// since it's used by Cartridge CLI to start and stop instance
```

Comment:
It could be mentioned in readme as well

Reply:
This is for developer to remember why PID files aren't deleted.

Reply:
I think it could be useful to know for user